nix/haskell.nix

```diff
@@ -1,16 +0,0 @@
-# This file is part of the IOGX template and is documented at the link below:
```

Comment:
@zeme-wana Why did you remove this? Seems like it's still part of IOGX

Reply:
Yes but the link is obsolete. 
Soon I'll be updating all the readmes in the repos using iogx so that they have up-to-date docs.